r-speedglm/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-speedglm
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.3_1"
+
+source:
+  fn: speedglm_0.3-1.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/speedglm_0.3-1.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/speedglm/speedglm_0.3-1.tar.gz
+
```

Comment:
No from this tarball which you want to download/compile. -> http://cran.r-project.org/src/contrib/speedglm_0.3-1.tar.gz